src/service/addToLike.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const addToLike = async (_id, token, postDispatch) => {
+  const response = await axios({
+    method: 'POST',
+    url: `/api/posts/like/${_id}`,
+    headers: { authorization: token },
+  });
+  postDispatch({ type: 'ADD_TO_LIKE', payload: response.data.posts });
```

Comment:
**must**: Try creating a constants object with all these statements to avoid any potential bugs.

Reply:
Yes, I did try to do this in the reducer so that I know exactly where the problem is.
But then it would become impure functions.
I will add constants to the reduce., Thank you, Aman.